Tools/.pre-commit-config.yaml

```diff
@@ -1,7 +1,7 @@
 default_stages: [commit]
 repos:
   - repo: git@github.cds.internal.unity3d.com:unity/gfx-sdet-tools.git
-    rev: 14104d8
+    rev: 259e643c92aa24f3536369ac5cb21f70b1071276
```

Comment:
TODO: Update pointer once https://github.cds.internal.unity3d.com/unity/gfx-sdet-tools/pull/70 is merged